package.json

```diff
@@ -14,5 +14,8 @@
         "redux-query": "3.3.1",
         "redux-query-interface-superagent": "3.3.1",
         "redux-query-react": "3.3.1"
+    },
+    "resolutions": {
+        "@types/react": "^17"
```

Comment:
A limitation of this is that `yarn` resolutions **must** be specified in the root package.json. So if we had differing versions of react (we do) AND multiple both versions used typescript (they don't) this could be problematic.